code/game/objects/structures/therapy.dm

```diff
@@ -154,7 +154,7 @@
 	if(!do_mob(user, H, 10 SECONDS))
 		return
 
-	if((!user in view(1,target)))
+	if(!(user in view(1, loc)))
```

Comment:
I'm not sure about this one, but I think it's right.

Reply:
Actually. Now that I think about it. An exact replacement would be `H`. The check is meant to look if the user has moved and is no longer in view of the target (`H`). I'm not really sure why this check exists since the `do_mob` is already there, though.